### DISCUSSION TITLE
part-ii/2017-129/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#398] [Question comments]
Part II 2017 Riemann Surfaces: Paper 3, Section II, F 

https://questions.tripos.org/part-ii/2017-129/

**Comment by artie2000** [comment]
<details>
<summary>Ramification of $\pi$</summary>

Ramified at the $n^\text{th}$ roots of unity $(\zeta_n^j,0)$, ramification index at each point is 2, branch points are $\zeta_n^j$.

</details>
<details>
<summary>Ramification of $\tau$</summary>

Ramified at $(0,\pm i)$, ramification index at each point is $n$, branch points are $\pm i$.

</details>
<details>
<summary>Genus of $\bar R$:</summary>

By valency theorem at ramification points, $\deg\bar\pi=2$. Use parity in Riemann-Hurwitz to determine the ramification at infinity.

$g(\bar R)=\begin{cases}\frac{n-2}{2}&n\text{ even}\\\frac{n-1}{2}&n\text{ odd}\end{cases}$

</details>
